### DISCUSSION TITLE
plainCookie dont working

### DISCUSSION THREAD

**Discussion by JonhnyDev** [adonisjs/core#2499] [Help] [answered]
let createcookie = request.plainCookie('hash_business', getBusiness.hash)
let getcookie= request.plainCookie('hash_business')

console.log('createcookie', createcookie)
console.log('getcookie', getcookie)

output:
createcookie 94jtjfigjdoifjgoidjfg
getcookie undefined

I tried to put: "let createcookie = request.plainCookie('hash_business', getBusiness.hash, {path: '/'})" unsuccessfully.

does anyone know what may be happening?

**Comment by thetutlage** [comment, marked as answer]
You set cookie using the `response` object. The `request` object is for reading the value. And also please share the framework version you are using, with every question you ask :)
